ide/coqide/ideutils.ml

```diff
@@ -174,6 +174,19 @@ let ulen uni_ch =
   else if uni_ch < 0x10000 then 3
   else 4
 
+(* workaround for lablgtk bug up to version 2.18.13 *)
+(* replaces: buffer#get_iter_at_byte ~line index *)
+(* see https://github.com/garrigue/lablgtk/pull/175 *)
```

Comment:
doesn't seem related, https://github.com/garrigue/lablgtk/pull/175 is about copy_string_check AFAICT

Reply:
It should be fixed. It is 181.

Reply:
Got it.